airbyte-integrations/bases/base-java/Dockerfile

```diff
@@ -1,5 +1,21 @@
 FROM openjdk:14.0.2-slim
 COPY --from=airbyte/integration-base:dev /airbyte /airbyte
+COPY --from=airbyte/normalization:dev /airbyte /airbyte
```

Comment:
I didn't find in the gradle tasks how to make the base-java:airbyteDocker depends on airbyte/normalization:airbyteDocker though...

Reply:
You need something like `implementation files(project(':airbyte-integrations:bases:base-normalization').airbyteDocker.outputs)` in gradle

Reply:
I've tried that but i really don't see why doesn't like it...
To me, there is no significant differences in the build.gradle between the other projects?

![Screenshot 2020-11-19 at 21 37 08](https://user-images.githubusercontent.com/7718175/99721411-770e4780-2aaf-11eb-8349-2477505deeef.png)